lib/vsc/administration/vo.py

```diff
@@ -211,7 +225,18 @@ def _create_fileset(self, filesystem_name, path, parent_fileset=None, fileset_na
 
         The parent_fileset is used to support older (< 3.5.x) GPFS setups still present in our system
         """
-        self.gpfs.list_filesets()
+        try:
```

Comment:
seem similar to the user code (a lot of this does).

Reply:
indeed, I made a new parent class for both `VscTier2AccountpageUser` and `VscTier2AccountpageVo` in https://github.com/hpcugent/vsc-administration/pull/134/commits/1f210803bd6c84daff6606f006e96b164bc7903f. It gathers all those methods that can be shared, if you see anything else similar between the two let me know.